daemon/src/main/java/org/mvndaemon/mvnd/daemon/Server.java

```diff
@@ -567,6 +567,15 @@ int getClassOrder(Message m) {
             return 3;
         case Message.MOJO_STARTED:
             return 4;
+        case Message.TRANSFER_INITIATED:
+        case Message.TRANSFER_STARTED:
+            return 40;
+        case Message.TRANSFER_PROGRESSED:
+            return 41;
+        case Message.TRANSFER_CORRUPTED:
+        case Message.TRANSFER_SUCCEEDED:
+        case Message.TRANSFER_FAILED:
+            return 42;
```

Comment:
At some occasion in the future, we could perhaps change the Message IDs to match this ordering so that we do not need an extra comparator method.